Makefile.js

```diff
@@ -527,7 +527,7 @@ target.lint = function() {
     }
 
     echo("Validating JavaScript test files");
-    lastReturn = exec(ESLINT + testCache + TEST_FILES);
+    lastReturn = exec(`${ESLINT}${testCache}"tests/**/*.js"`);
```

Comment:
Good catch. Seems kinda strange to me that we don't use `TEST_FILES` anymore, though. Isn't there a way we can fix the `TEST_FILES` variable to account for `tests/conf` too?